README.md

```diff
@@ -32,4 +32,44 @@ docker run -it yast-rmt-image rspec
 
 ### Package 
 
-The package gets built for SLE15 here: https://build.opensuse.org/package/show/systemsmanagement:SCC:RMT/yast2-rmt
+The package gets built for SLE 15 and openSUSE Leap 15 here: 
+https://build.opensuse.org/package/show/systemsmanagement:SCC:RMT/yast2-rmt
+
+
+#### Submit Requests to openSUSE Factory and SLES
+
+To get a maintenance request accepted, each changelog entry needs to reference a bug or feature
+request with `bsc#123` or `fate#123`.
+
+##### Factory First
+
+To submit a request to openSUSE Factory, issue this commands in the console:
+
+```bash
+osc sr systemsmanagement:SCC:RMT yast2-rmt openSUSE:Factory --no-cleanup
+```
+
+
+##### Internal Build Service
```

Comment:
Why would we ever need to submit to IBS? (Considering that it is a linked project from OBS)